### ISSUE TITLE
Email warning

### ISSUE THREAD

**Issue by blackrosezy** [nicolargo/glances#475]
I just found this project and like it very much. It is nice if we can set SMTP server and warning will be automatically send to an email.

An example:
If ongoing warning is longer than 60 seconds, send to user email
```
--warning-threshold-time=60s --smtp-user=test --smtp-pass=abc123 --stmp-host=example.com --warning-email=alert@example.com
```

**Comment by nicolargo** [comment]
Hi, 

the action on trigger feature will be implemented in the 2.3 release (actualy DEVELOP branch).
See: https://github.com/nicolargo/glances/issues/132

With this feature, you will be able to run an external script to generate mail or what ever you want.

Nicolas

**Comment by nicolargo** [comment]
REJECTED: cause duplicated https://github.com/nicolargo/glances/issues/132
